lib/jekyll-avatar.rb

```diff
@@ -6,10 +6,29 @@ module Jekyll
   class Avatar < Liquid::Tag
     include Jekyll::LiquidExtensions
 
+    def self.generate_template_with(keys)
+      attrs = (BASE_ATTRIBUTES + keys).map! { |key| %(#{key}="%<#{key}>s") }.join(" ")
+      "<img #{attrs} />"
+    end
+    private_class_method :generate_template_with
+
+    #
+
     SERVERS      = 4
     DEFAULT_SIZE = 40
     API_VERSION  = 3
 
+    BASE_ATTRIBUTES = %w(
+      class alt width height data-proofer-ignore src
+    ).freeze
+
+    BASE_TEMPLATE = generate_template_with %w(srcset)
+    LAZY_TEMPLATE = generate_template_with %w(data-src data-srcset)
+
+    private_constant :BASE_ATTRIBUTES, :BASE_TEMPLATE, :LAZY_TEMPLATE
+
+    #
```

Comment:
Looks like a stray comment snuck in.

Reply:
It was intentionally placed to improve readability. But since you asked about it, I guess it is more of a distraction instead.